components/discover/defiTable.tsx

```diff
@@ -305,14 +312,33 @@ const DataTable: FunctionComponent<DataTableProps> = ({ data, loading }) => {
   return (
     <div className="w-full overflow-x-auto">
       <div className="">
-        <div className={`flex w-100 flex-col gap-2`}>
-          <Typography type={TEXT_TYPE.H4} color="secondary">
-            Explore reward opportunities
-          </Typography>
-          <Typography type={TEXT_TYPE.BODY_MICRO} color="secondary">
-            Find the best opportunities, and earn tokens
-          </Typography>
+        <div className={`flex w-100 lg:flex-row flex-col justify-between items-start`}>
+          <div className={`flex w-100 flex-col gap-2`}>
+            <Typography type={TEXT_TYPE.H4} color="secondary">
+              Explore reward opportunities
+            </Typography>
+            <Typography type={TEXT_TYPE.BODY_MICRO} color="secondary">
+              Find the best opportunities, and earn tokens
+            </Typography>
+          </div>
+          <div
+            onClick={() => setShowClaimModal(true)}
+            className="flex flex-row items-center justify-evenly gap-4 bg-white rounded-lg modified-cursor-pointer h-min lg:mt-2 mt-8 px-4 lg:py-1 py-2">
+            <AppIcon app="starknet" />
+            <Typography type={TEXT_TYPE.BUTTON_LARGE} color="background">
+              Claim all
```

Comment:
The icon is smaller on the design, and the padding is bigger:
<img width="408" alt="image" src="https://github.com/user-attachments/assets/b3e1a6e1-8c22-4b92-84d6-07f62d1a0bfb">

Reply:
Just touched it a little bit, let me know if it goes well with the design:
```js
<div 
  onClick={() => setShowClaimModal(true)}
  className="flex flex-row items-center justify-evenly gap-4 bg-white rounded-xl modified-cursor-pointer h-min lg:mt-2 mt-8 px-6 py-2.5"
>
  <AppIcon app="starknet" className="w-5 h-5" />
  <Typography type={TEXT_TYPE.BUTTON_LARGE} color="background">
    Claim all
  </Typography>
</div>
```
![image](https://github.com/user-attachments/assets/41e989f1-8ffb-4462-9193-bfb9fd0d047a)